### ISSUE TITLE
Something wrong in the marker explorer for the adipogenicity project

### ISSUE THREAD

**Issue by tetomonti** [montilab/Xposome#15]
When I select a gene there (e.g., pparg), the 2nd density plots shows three categories, vehicle, suspected, and "yes". The three categories should be "no", "suspected", and "yes" (possibly, with no and yes replaced with more intuitive labels, or at least the legend title should be "PPARg modifier?".
And the corresponding table lists only few vehicle rows and then suspected and yes only. The original dataset contained very few vehicles (i.e., DMSO-treated), and then treated with known PPARg modifiers and non-modifiers, plus suspected ones.
See [EHP manuscript](https://ehp.niehs.nih.gov/doi/10.1289/EHP6886):
"A classification model was inferred from the training set consisting of 37 known PPARγ-modifying compounds and 23 known non-PPARγ modifying compounds, including vehicle, to predict the labeling of the test set of 17 potential PPARγ-modifying compounds (Table S1)."

**Comment by tetomonti** [comment]
We ascertained that the source of the problem is that currently only the known (n=37) and potential (n=17) adipogens plus vehicle are included in the list.
The dataset should be updated to include all the chemicals (i.e., add the 23 non-adipogens). See the K2T tab for the full set.
